src/main/java/org/opensearch/geospatial/action/geojson/upload/UploadGeoJSONTransportAction.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.action.geojson.upload;
+
+import org.opensearch.action.ActionListener;
+import org.opensearch.action.support.ActionFilters;
+import org.opensearch.action.support.HandledTransportAction;
+import org.opensearch.action.support.master.AcknowledgedResponse;
+import org.opensearch.common.inject.Inject;
+import org.opensearch.tasks.Task;
+import org.opensearch.transport.TransportService;
+
+/**
+ * TransportAction to handle import operation
+ */
+public class UploadGeoJSONTransportAction extends HandledTransportAction<UploadGeoJSONRequest, AcknowledgedResponse> {
+
+    @Inject
+    public UploadGeoJSONTransportAction(TransportService transportService, ActionFilters actionFilters) {
+        super(UploadGeoJSONAction.NAME, transportService, actionFilters, UploadGeoJSONRequest::new);
+    }
+
+    @Override
+    protected void doExecute(Task task, UploadGeoJSONRequest request, ActionListener<AcknowledgedResponse> actionListener) {
+        // TODO Add logic to execute import action
```

Comment:
Can we add reference to logic implementation issue with this TODO?

Reply:
That is going to be my next PR. Do you prefer to create an issue?

Reply:
If it's immediate follower type of change I'm fine with existing TODO. I was thinking of a case when such change is  delayed for unspecified date.

Reply:
i understand. I updated TODO with github issue.